ingestion/src/metadata/ingestion/source/dashboard/redash.py

```diff
@@ -94,24 +94,31 @@ def yield_dashboard(
         """
         Method to Get Dashboard Entity
         """
+        try:
+            dashboard_description = ""
+            for widgets in dashboard_details.get("widgets", []):
+                dashboard_description = widgets.get("text")
+            yield CreateDashboardRequest(
+                name=dashboard_details.get("id"),
+                displayName=dashboard_details["name"],
+                description=dashboard_description,
+                charts=[
+                    EntityReference(id=chart.id.__root__, type="chart")
+                    for chart in self.context.charts
+                ],
+                service=EntityReference(
+                    id=self.context.dashboard_service.id.__root__,
+                    type="dashboardService",
+                ),
+                dashboardUrl=f"/dashboard/{dashboard_details.get('slug', '')}",
+            )
+            self.status.scanned(dashboard_details["name"])
 
-        dashboard_description = ""
-        for widgets in dashboard_details.get("widgets", []):
-            dashboard_description = widgets.get("text")
-        yield CreateDashboardRequest(
-            name=dashboard_details.get("id"),
-            displayName=dashboard_details["name"],
-            description=dashboard_description,
-            charts=[
-                EntityReference(id=chart.id.__root__, type="chart")
-                for chart in self.context.charts
-            ],
-            service=EntityReference(
-                id=self.context.dashboard_service.id.__root__, type="dashboardService"
-            ),
-            dashboardUrl=f"/dashboard/{dashboard_details.get('slug', '')}",
-        )
-        self.status.scanned(dashboard_details["name"])
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.warning(
+                f"Error to creating dashboard for {dashboard_details}: {exc}"
```

Comment:
Please change the message to `Error to yield dashboard for {dashboard_details}: {exc}`